app/src/main/java/org/qosp/notes/ui/MainActivity.kt

```diff
@@ -210,7 +215,30 @@ class MainActivity : BaseActivity() {
 
     private fun setupNavigation() {
         fun createNotebookMenuItems(notebooks: List<Notebook>) {
-            notebooks.forEach { notebook ->
+
+            // Obtaining the current setting of notebook sorting order.
+            val sort = runBlocking {
+                return@runBlocking async {
+                    preferenceRepository
+                        .getAll()
+                        .map { it.sortNavdrawerNotebooksMethod }
+                        .first()
+                        .name
+                }.await()
+            }
+
+            // Sorting the notebooks.
+            var sortedNotebooks: List<Notebook>
+            when (sort) {
```

Comment:
Combine these two by:
```suggestion
            val sortedNotebooks = when (sort) {
```
This can also make it val instead of var

Reply:
nit: You can add a static import for `SortNavdrawerNotebooksMethod` for readability

Reply:
Done. Carried out in f949e9c4118a72110ecfe55d7de3fbf63caaedbf.